includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -205,6 +214,27 @@ public static function render_rfc9421_signature_field() {
 		<?php
 	}
 
+	/**
+	 * Render show following UI field.
+	 */
+	public static function render_following_ui_field() {
+		$value = \get_option( 'activitypub_following_ui', '0' );
+		?>
+		<p>
+			<label>
+				<input type="checkbox" id="activitypub_following_ui" name="activitypub_following_ui" value="1" <?php checked( '1', $value ); ?> />
+				<?php \esc_html_e( 'Display the "Following" interface in the admin menus and settings.', 'activitypub' ); ?>
```

Comment:
Given the temporary nature of this option, should we keep these strings untranslated and lower the work load of translators?